src/Lecture1.hs

```diff
@@ -77,7 +88,12 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+
+minmax :: Int -> Int -> Int -> Int
+sortAsc :: [Int] -> [Int]
+
+sortAsc l =  Data.List.sort l
+minmax x y z = last (sortAsc [x, y, z]) - head (sortAsc [x, y, z])
```

Comment:
That's a neat implementation 😎 
Alternatively, you can use standard functions `maximum` and `minimum` that work with lists 👍🏻 
With this approach, you won't need sorting. Also, you can remove the `Data.List` import and all GHC warnings associated with it 🧹